ApplicationManager.js

```diff
@@ -183,16 +183,18 @@ class ApplicationManager {
   resetParametersButtons() {
     d3.select("#algoParamsButtonSpace").selectAll("*").remove();
     algorithmsConfig[this.algo.getName()].parameters.forEach(param => {
-      const { input_type: inputType } = paramsConfig[param];
-      const { domain, get_init_value: getInitValue } = paramsConfig[param].values[this.plot.getType()];
-      let initValue;
-      if (param === paramNames.objectiveFunction) {
-        initValue = this.objectiveFunction;
-      } else {
-        initValue = this.algoParams[param];
+      if (!(param === paramNames.x_ini)) {
```

Comment:
It works in this case because x_ini is the only param to have a click input but I think the following is what we actually want:
```suggestion
      if (paramsConfig[param].input_type !== inputTypes.click) {
```